.gitlab/pipeline/short-benchmarks.yml

```diff
@@ -47,7 +47,7 @@ short-benchmark-westend: &short-bench
   tags:
     - benchmark
   script:
-    - ./artifacts/polkadot-parachain benchmark pallet --chain $RUNTIME_CHAIN --pallet "*" --extrinsic "*" --steps 2 --repeat 1
+    - ./artifacts/polkadot-parachain-omni-node benchmark pallet --chain $RUNTIME_CHAIN --pallet "*" --extrinsic "*" --steps 2 --repeat 1
```

Comment:
@ggwpez the days for this command is also numbered, is there a ticket to move this to the omni-bencher? 

Side question: can the omni-node do benchmarking as well? what are the runtime-dependent parts of the benchmarking code that live in the node side? I suppose there are dependencies that led to the omni-bencher being created, but idk what they are.